pkg/tnf/identifier/identifiers.go

```diff
@@ -199,3 +201,15 @@ var VersionIdentifier = Identifier{
 	URL:             versionIdentifierURL,
 	SemanticVersion: versionOne,
 }
+
+// CnfFsDiffIdentifier is the Identifier used to represent the generic cnf_fs_diff test.
+var CnfFsDiffIdentifier = Identifier{
+	URL:             cnfFsDiffURL,
+	SemanticVersion: versionOne,
+}
+
+// ContainerIDIdentifier is the Identifier used to represent the generic cnf_fs_diff test.
+var ContainerIDIdentifier = Identifier{
+	URL:             containerIDURL,
+	SemanticVersion: versionOne,
+}
```

Comment:
Please add these to the Catalog:
https://github.com/test-network-function/test-network-function/blob/main/pkg/tnf/identifier/identifiers.go#L73